packages/cache/src/LocalCacheStorage.ts

```diff
@@ -14,45 +13,28 @@ export class LocalCacheStorage extends CacheStorage {
     return path.join(this.internalCacheFolder, hash);
   }
 
-  protected async _fetch(
-    hash: string,
-    outputFolder: string | string[]
-  ): Promise<boolean> {
+  protected async _fetch(hash: string): Promise<boolean> {
     const localCacheFolder = this.getLocalCacheFolder(hash);
 
     if (!fs.pathExistsSync(localCacheFolder)) {
       return false;
     }
 
-    await Promise.all(
-      outputFolderAsArray(outputFolder).map(async folder => {
-        const locationInCache = path.join(localCacheFolder, folder);
-
-        if (!fs.pathExistsSync(locationInCache)) {
-          return;
-        }
-
-        await fs.mkdirp(folder);
-        await fs.copy(locationInCache, folder);
-      })
-    );
+    fs.readdirSync(localCacheFolder).forEach(fileOrFolder => {
```

Comment:
Will `fileOrFolde` be absolute paths, or relative to `localCacheFolder`? If it is absolute, then the `path.join` wouldn't work below.

Reply:
the path provided by fast-glob is relative